test-data/unit/check-expressions.test

```diff
@@ -1688,6 +1688,22 @@ main:1: note: Revealed type is "Any"
 def reveal_type(x: int) -> None: pass
 reveal_type("foo") # E: Argument 1 to "reveal_type" has incompatible type "str"; expected "int"
 
+[case testTypingRevealType]
+from typing import reveal_type
+from typing import reveal_type as show_me_the_type
```

Comment:
optional nit: Let's also try `rt = reveal_type` and `rt(1)`

Reply:
That actually doesn't work, probably because at the place in semanal where we recognize reveal_type we can't see through assignments yet. Fixing it would probably be a lot more complicated than this PR, so if we think it's worth fixing, we should do it separately. It doesn't seem too important as no user has asked for it so far.